middle_end/flambda2.0/terms/flambda_primitive.ml

```diff
@@ -291,18 +291,8 @@ let print_equality_comparison ppf op =
   | Eq -> Format.pp_print_string ppf "Eq"
   | Neq -> Format.pp_print_string ppf "Neq"
 
-type is_safe = Safe | Unsafe
-
-let print_is_safe ppf is_safe =
-  match is_safe with
-  | Safe -> Format.pp_print_string ppf "Safe"
-  | Unsafe -> Format.pp_print_string ppf "Unsafe"
-
-let compare_is_safe is_safe1 is_safe2 =
-  Stdlib.compare is_safe1 is_safe2
-
 type bigarray_kind =
-  | Unknown
+  (* | Unknown *)
```

Comment:
Can these commented lines be deleted?  If they should remain for the moment please add a comment explaining why.